pkg/controller/service/service_controller.go

```diff
@@ -139,15 +145,23 @@ func (r *ReconcileService) Reconcile(request reconcile.Request) (reconcile.Resul
 	return reconcile.Result{}, nil
 }
 
-const containerJFRPort int = 9091 // TODO make a property in ContainerJFR CRD?
-
 func isJFRAwareService(svc *corev1.Service) bool {
+	_, err := getServiceJMXPort(svc)
+	return err == nil
+}
```

Comment:
Perhaps we should get rid of this function and just retain the JMX port number (if any), then we wouldn't have to call `getServiceJMXPort` again when creating the FlightRecorder object.

Reply:
Makes sense, I'll do that